packages/flame/src/Alert/AlertInCard.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import css from '@styled-system/css';
+
+import { AlertIcons } from './AlertIcons';
+import { CloseButton } from './CloseButton';
+import { Flex } from '../Core';
+
+interface Props {
+  type?: string;
+  isHidden?: string;
+  noCloseBtn?: boolean;
+  /** Function called when Close button is tapped */
+  onClose?: Function;
+}
+const AlertInCard: React.FC<Props> = ({
+  type = 'info',
+  noCloseBtn = false,
+  onClose,
+  children,
+  ...restProps
+}) => {
+  const [isHidden, setIsHidden] = React.useState(false);
+
+  const handleClose = (e: React.MouseEvent<HTMLElement>) => {
+    if (onClose && typeof onClose === 'function') onClose(e);
+    setIsHidden(s => !s);
+  };
+
+  if (isHidden) return null;
+
+  return (
+    <Flex
+      css={css({
+        p: 3,
+        alignItems: 'center',
+        borderRadius: 'radius-2',
+        border: '1px solid',
+        variant: `alertInCardVariants.${type}`,
+      })}
+      {...restProps}
+    >
+      <Flex className="fl-alert__icon" flex={0} mt="-1px">
+        <AlertIcons type={type} />
+      </Flex>
+      <div css={css({ flex: '1', pl: 2, fontSize: ['text', 'text-s'] })}>{children}</div>
```

Comment:
Why a `div` instead of `Text`? I would also add the corresponding `lineHeight` depending on size: https://design.lightspeedhq.com/components/Text/#text

Reply:
I've swapped it over to `<Text />` + manually setting font size and line height to be legit .

Having said that, this could've practically also been a `p` tag and we'd be fine. Text isn't particularly useful anymore, at least not in its current state. The `size` props does not define the right values and doesn't even scale for mobile use. Ideally, Text should've taken care of this particular scenario for us.

The problem is, in abstract, the font size values are different than what is currently available in the theme file. We'll have to figure out how to gracefully move to the newer values. Text values for desktop should've been 14px and mobile 16px.

But the theme or the component does not define this (we're still 16px on desktop and 16px in mobile). Discussion for another time tho